internal/app/cli.go

```diff
@@ -160,6 +155,23 @@ func getClusterSourceFlags(config *CliConfig) []cli.Flag {
 	}
 }
 
+func getOutputSourceFlags(config *CliConfig) []cli.Flag {
+	return []cli.Flag{
+		&cli.BoolFlag{
+			Name:        "silent",
+			Aliases:     []string{"s"},
+			Usage:       "",
```

Comment:
I know it was empty before but if you can add some description by this chance it will be great :-)